include/pka_config.h

```diff
@@ -53,13 +53,10 @@
 // and ring spacing.

```

Comment:
definitions at line 40 and 42 must change as well

Reply:
For BF1 & BF2 these will not change right ? do you mean for BF3? If so I'll be taking care of that separately. This change is only for BF1 & BF2.